video/out/vo.c

```diff
@@ -1088,7 +1112,12 @@ static void *vo_thread(void *ptr)
             break;
         stats_event(in->stats, "iterations");
         vo->driver->control(vo, VOCTRL_CHECK_EVENTS, NULL);
-        bool working = render_frame(vo);
+        bool working = false;
+        if (vo->force_drop && vo->force_drops_allowed) {
+            drop_unrendered_frame(vo);
```

Comment:
I don't understand this. vo_vdpau can drop frames on flip_page, and it uses vo_increment_drop_count for that.